### PR TITLE
Added steps for installing opa in container file and updated README

### PR DESCRIPTION
**Pull request by jthompkins**
Adding container file for automated governance image that contains both rekor and opa.

### CODE REVIEW

File: ploigos-tool-autogov/Containerfile.ubi8 [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    name="ploigos/ploigos-tool-openscap" \
+    summary="$DESCRIPTION" \
+    description="$DESCRIPTION" \
+    License="GPLv2+" \
+    architecture="x86_64" \
+    io.k8s.display-name="Ploigos - Tool - Rekor" \
+    io.k8s.description="$DESCRIPTION" \
+    io.openshift.expose-services="" \
+    io.openshift.tags="ploigos,rekor" \
+    com.redhat.component="ploigos-tool-rekor-container"
```
Comment (gfulton-redhat) [inline ploigos-tool-autogov/Containerfile.ubi8:19]: The component should reference `ploigos-tool-autogov`?

```suggestion
    com.redhat.component="ploigos-tool-autogov"
```

**Review by gfulton-redhat — CHANGES_REQUESTED** [review]
@jthompkins you also need to update the README with a section that describes `ploigos-tool-autogov`.

File: ploigos-tool-autogov/Containerfile.ubi8 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    maintainer="Ploigos <ploigos@redhat.com>" \
+    name="ploigos/ploigos-tool-openscap" \
+    summary="$DESCRIPTION" \
+    description="$DESCRIPTION" \
+    License="GPLv2+" \
+    architecture="x86_64" \
+    io.k8s.display-name="Ploigos - Tool - Rekor" \
+    io.k8s.description="$DESCRIPTION" \
+    io.openshift.expose-services="" \
+    io.openshift.tags="ploigos,rekor" \
```
Comment (gfulton-redhat) [inline ploigos-tool-autogov/Containerfile.ubi8:18]:
```suggestion
    io.openshift.tags="ploigos,autogov" \
```

File: ploigos-tool-autogov/Containerfile.ubi8 (hunk 1 above)
Comment (gfulton-redhat) [inline ploigos-tool-autogov/Containerfile.ubi8:19]:
```suggestion
    com.redhat.component="ploigos-tool-autogov"    
```

Comment (jthompkins) [reply ploigos-tool-autogov/Containerfile.ubi8]: Changed

File: ploigos-tool-autogov/Containerfile.ubi8 (hunk 2 above)
Comment (jthompkins) [reply ploigos-tool-autogov/Containerfile.ubi8]: Changed

**Review by gfulton-redhat — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -71,13 +71,22 @@ Defines a maven container image. Built from `ploigos-base-java-8`.
 Defines an OpenSCAP based image scanning container image. Built from ploigos-base.
 
 ## ploigos-tool-rekor
```
Comment (itewk) [inline README.md:73]: as talked about, nuke the rekor image and we will jut have the autogov image going forward

File: README.md [hunk 4]
```diff
@@ -123,7 +132,9 @@ to kick off which will build, test, publish the images (that can) to [quay.io/pl
   * ubi8
 * ploigos-tool-openscap
   * ubi8
-* ploigos-tool-reokor
+* ploigos-tool-rekor
```
Comment (itewk) [inline README.md:135]: as talked about, nuke the rekor image and we will jut have the autogov image going forward

File: README.md [hunk 5]
```diff
@@ -146,6 +157,7 @@ podman build --tag ploigos-tool-helm        --build-arg BASE_IMAGE=ploigos-base
 podman build --tag ploigos-tool-maven       --build-arg BASE_IMAGE=ploigos-base-java-8     ploigos-tool-maven
 podman build --tag ploigos-tool-openscap    --build-arg BASE_IMAGE=ploigos-tool-containers ploigos-tool-openscap
 podman build --tag ploigos-tool-rekor       --build-arg BASE_IMAGE=ploigos-base            ploigos-tool-rekor
```
Comment (itewk) [inline README.md:159]: as talked about, nuke the rekor image and we will jut have the autogov image going forward

**Review by itewk — CHANGES_REQUESTED** [review]

File: .github/workflows/publish.yaml [hunk 6]
```diff
@@ -1152,6 +1152,110 @@ jobs:
...
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+
+      - name: Test Image 🧪
+        run: |
+          echo "Check rekor binary exists"
+          docker run -u 1001 --entrypoint='' ${{ env.IMAGE_TAG_LOCAL }} /bin/sh -c "rekor --help"
+      - name: Test Image
+        run: |
+          echo "Check opa binary exists"
```
Comment (itewk) [inline .github/workflows/publish.yaml:1235]: merge this into the previous test image step, rather then having multiple steps just do all the calls one after the other.

File: ploigos-tool-autogov/Containerfile.ubi8 [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    chmod 775 /usr/bin && \
+    chmod 777 /usr/local/bin/rekor && \
+    chown 1001:0 /usr/local/bin/rekor
+
+#Install opa
+RUN curl -L -o opa https://openpolicyagent.org/downloads/v0.29.4/opa_linux_amd64 && \
+    chmod 775 ./opa && \
+    mv opa /usr/bin/
+
+
```
Comment (itewk) [inline ploigos-tool-autogov/Containerfile.ubi8:50]: nit: remove extra new line

File: ploigos-tool-autogov/README.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+# ploigos-tool-autogov
+
+This repository contains the container definition for creating the Ploigos tool for running rekor and opa commands.
```
Comment (itewk) [inline ploigos-tool-autogov/README.md:3]: nit: id rephrase this to say "this repoisotry contains the container definition for creating the Ploigos tools container for performing auto governance oeprations. Included tools are:
* rekor
* opa

File: ploigos-tool-autogov/contrib/centos.repo [hunk 9]
```diff
@@ -0,0 +1,5 @@
+[centos]
```
Comment (itewk) [inline ploigos-tool-autogov/contrib/centos.repo:1]: @jthompkins what is this needed for?

**Review by itewk — CHANGES_REQUESTED** [review]

File: ploigos-tool-autogov/contrib/centos.repo (hunk 9 above)
Comment (jthompkins) [reply ploigos-tool-autogov/contrib/centos.repo:1]: I am not sure actually. I copied the ploigos-tool-rekor directory as a starting point and it had this file already

File: README.md (hunk 3 above)
Comment (jthompkins) [reply README.md:73]: Nuked.

File: README.md (hunk 4 above)
Comment (jthompkins) [reply README.md]: Nuked.

File: README.md (hunk 5 above)
Comment (jthompkins) [reply README.md]: Nuked.

File: .github/workflows/publish.yaml [hunk 10]
```diff
@@ -1152,6 +1152,110 @@ jobs:
...
+            org.opencontainers.image.version=${{ steps.prep.outputs.version }}
+            org.opencontainers.image.revision=${{ github.sha }}
+            org.opencontainers.image.licenses=${{ github.event.repository.license.name }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache
+
+      - name: Test Image 🧪
+        run: |
+          echo "Check rekor binary exists"
+          docker run -u 1001 --entrypoint='' ${{ env.IMAGE_TAG_LOCAL }} /bin/sh -c "rekor --help"
```
[the remark this answers is not in the document]
Comment (jthompkins) [reply .github/workflows/publish.yaml]: Fixed.

File: .github/workflows/publish.yaml (hunk 6 above)
Comment (jthompkins) [reply .github/workflows/publish.yaml]: Fixed.

File: ploigos-tool-autogov/Containerfile.ubi8 (hunk 7 above)
Comment (jthompkins) [reply ploigos-tool-autogov/Containerfile.ubi8]: Fixed.

**Review by itewk — APPROVED** [review]
